src/CoinbasePro.ts

```diff
@@ -40,7 +40,7 @@ export class CoinbasePro {
     SANDBOX: ClientConnection;
   } = {
     PRODUCTION: {
-      REST: 'https://api.pro.coinbase.com',
+      REST: 'https://api.exchange.coinbase.com',
```

Comment:
Semi-important note here - it looks like Coinbase is defaulting to using 'api.exchange' instead of 'api.pro' as their preferred base url.

Reply:
Nice find! 👍

Reply:
Good catch! They made this change in June last year: https://docs.cloud.coinbase.com/exchange/docs/changelog#2021-jun-03